video/out/opengl/video.c

```diff
@@ -3006,13 +3006,13 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
     if (!has_frame || !mp_rect_equals(&p->dst_rect, &target_rc)) {
         struct m_color c = p->clear_color;
         float color[4] = {c.r / 255.0, c.g / 255.0, c.b / 255.0, c.a / 255.0};
-        p->ra->fns->clear(p->ra, target.tex, color, &target_rc);
+        p->ra->fns->clear(p->ra, target.tex, color);
```

Comment:
To be clear, I was proposing that we remove both clear() and clear_rect() from RA, make the “clear as part of a renderpass” API the only way to clear something, and then maybe support “dummy renderpasses” (which would skip shader compilation or something) so you can do your clearing-outside-of-render-pass? Not sure..

I still think the easiest way to make everybody happy would be to just go back to having a single clear(mp_rect) RA API call and make ra_vk translate this to per-renderpass clear calls under the hood.